pom.xml

```diff
@@ -306,7 +306,7 @@
 					<dependency>
 						<groupId>com.pinterest</groupId>
 						<artifactId>ktlint</artifactId>
-						<version>0.49.1</version>
+						<version>0.50.0</version>
```

Comment:
Burde denne generere en haug med endringer, eller tas det i en egen PR ?

Reply:
Det er for en gang skyld ikke noe i denne versjon som fører til endringer i koden